apel/parsers/lsf.py

```diff
@@ -126,9 +126,10 @@ def parse(self, line):
             data[key] = mapping[key](items)
 
         # Input checking
-        if data['CpuDuration'] < 0:
+        # TODO: Check needed, across codebase: When testing try to compare with schema
```

Comment:
Open an issue rather than add a TODO to the codebase.

Reply:
I could have given a heads-up. TODO is the activity for my testing with rocky8 and 9 ticket. I will fix those as part of that. I can assure it is not major one. It is doable in very less time, Greg.  I will remove those eventually. Make sense ?